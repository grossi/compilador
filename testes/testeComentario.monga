/* asd */
int /* asd */a/* asd */, b, d, f;

int c;