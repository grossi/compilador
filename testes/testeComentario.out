program
 declista
  Declaracao Var
   tipo: int
   Lista de Nomes a
