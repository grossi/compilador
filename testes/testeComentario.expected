program
 declista
  Declaracao Var ID : f 
   tipo: int
   Declaracao Var ID : d 
    tipo: int
    Declaracao Var ID : b 
     tipo: int
     Declaracao Var ID : a 
      tipo: int
  declista
   Declaracao Var ID : c 
    tipo: int
